docs/Manpage.md

```diff
@@ -233,6 +233,10 @@ For tarballs, also print SHA-256 checksums.
 * `--force-bottle`:
   Download a bottle if it exists for the current or newest version of macOS, even if it would not be used during installation.
 
+### `formulae`
+
+Prints a sorted list of locally available formulas including shortnames.
```

Comment:
1. Use the root form of the verb "print" -- "Print a sorted list..."
2. Homebrew uses British English, so... formulae.
3. I'm not sure "shortname" is a word. Did you mean "... including their short names"?

Reply:
I'd say "output" instead of "print". Agreed on short name (have made that point above, I think).

Reply:
In my writing I've been using "output" only as a noun, preferring either "show" (the default), "display" (for single line output), "list" (for lists), "print" or "include" (for things that only slightly change or append to the output). So in this case, I'd write "List all locally available formulae including short names."